src/buza/static/buza/css/questions.css

```diff
@@ -75,3 +67,37 @@ textarea[name=body] {
     margin:1%;
 
 }
+
+.question__form_container{
+    width: 100%;
+    padding: 5% 5%;
+    margin: 8px 0;
+    text-align: center;
+    box-sizing: border-box;
+}
+
+.question__form_helptext{
+    opacity: 0.5;
+    font-size: smaller;
+
+}
+
+input[type=text], input[type=number], select, textarea {
+    border: 2px solid #32CD32;
+    width: 80%;
+    border-radius: 4px;
+    background-color: white;
+    padding: 8px 10px;
+}
+@media only screen and (max-width: 600px) {
+    body {
+        width:100%;
+        padding: 0px 0px;
+    }
+}
```

Comment:
What's the intent of this media query?

Reply:
the text looked messy when the device got smaller

Reply:
Hmm, was it actually due to this? I did some testing, and this shouldn't actually be changing or affecting anything visible…

If we can narrow down the actual problem, we can probably apply a more targeted fix.